spec/controllers/admin/consignments_controller_spec.rb

```diff
@@ -190,4 +190,33 @@
       end
     end
   end
+
+  describe '#update' do
+    let(:consignment) do
+      Fabricate(
+        :partner_submission,
+        state: 'open',
+        submission: Fabricate(:submission, state: 'approved'),
+        partner: Fabricate(:partner, name: 'Gagosian Gallery'),
+        sale_price_cents: 100_000
+      )
+    end
+
+    let(:offer) do
+      Fabricate(
+        :offer,
+        state: 'sent', partner_submission: consignment, offer_type: 'purchase'
+      )
+    end
+
+    before { consignment.update!(accepted_offer: offer) }
+
+    it 'sets offer state to draft if consignment state is cancelled' do
+      put :update,
+          params: {
+            id: consignment.id, partner_submission: consignment, as: :json
```

Comment:
@jonallured - I can't for the life of me figure out the correct way to pass this across:
```ruby
 1) Admin::ConsignmentsController#update sets offer state to draft if consignment state is cancelled
     Failure/Error:
       params.require(:partner_submission).permit(
         :canceled_reason,
         :currency,
         :sale_price_dollars,
         :sale_name,
         :sale_date,
         :sale_location,
         :sale_lot_number,
         :state,
         :partner_commission_percent_whole,

     NoMethodError:
       undefined method `permit' for "1":String
     # ./app/controllers/admin/consignments_controller.rb:107:in `consignment_params'
     # ./app/controllers/admin/consignments_controller.rb:82:in `update'
     # ./spec/controllers/admin/consignments_controller_spec.rb:215:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:29:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:29:in `block (2 levels) in <top (required)>'
```

Reply:
Yeah, happy to pair this afternoon. 👍